pkg/apis/core/v1alpha1/disable_types.go

```diff
@@ -18,10 +18,27 @@ type ConfigMapDisableSource struct {
 }
 
 type DisableStatus struct {
-	// Whether this is currently disabled.
+	// Whether this is currently disabled. Deprecated in favor of `State`.
 	Disabled bool `json:"disabled" protobuf:"varint,1,opt,name=disabled"`
```

Comment:
I'm guessing it's not worth the effort of v1alpha2ing everything to remove this

Reply:
ya, let's just keep them both for now